### DISCUSSION TITLE
Developer Toolkits

### DISCUSSION THREAD

**Discussion by HyperCrowd** [PsySecGroup/foundation#5] [Ideas]
To make this community really accelerated, lets find tools that allow people to quickly ramp up resources for data gathering, programming, and hosting

**Comment by HyperCrowd** [comment]
* [FlatFile.com](https://flatfile.com): Accept customer data from anywhere
* [Render.com](https://render.com)
  * Free hosting (100GBs/month)
  * Free web service (750 hours)
  * Free PostGres (3 months)
  * Free Redis (25MB)

**Comment by HyperCrowd** [comment]
[Dockerfile for Current NodeJS](https://github.com/nodejs/docker-node/blob/4e8fe34165d79044d7ea909021ccc0de3e3b4c6d/18/alpine3.14/Dockerfile):
```dockerfile
# specify the node base image with your desired version node:<version>
FROM node:16
# replace this with your application's default port
EXPOSE 8888
```

[Dockerfile for Current Python](https://github.com/docker-library/python/blob/5e580185c613edaf930fa6d23358235b950da0bd/3.11-rc/bullseye/Dockerfile):
```dockerfile
FROM python:3

WORKDIR /usr/src/app

COPY requirements.txt ./
RUN pip install --no-cache-dir -r requirements.txt

COPY . .

CMD [ "python", "./your-daemon-or-script.py" ]
```

[Dockerfile for current MariaDB](https://github.com/MariaDB/mariadb-docker/blob/c0d07be9ad5eb3bc212c6805cc8031308c56e9b6/10.9/Dockerfile) and please read the DockerHub instructions to set it up correctly.

[Docker for SQLite](https://hub.docker.com/r/keinos/sqlite3)

**Comment by HyperCrowd** [comment]
Yet another selenium webdriver wrapper API in python, with the aims of being the easist to use with only two lines of code to get running.

Unlike other selenium web driver wrappers, this one has tests for Windows10/MacOS/Ubuntu platforms.

One function is provided, open_webdriver(...) which takes care of downloading, installing and then running selenium.

Additionally, sane defaults are set, such as headless by default and ssl certs turned off. This code is also tested and examples are provided for use with the nuitka cross compiler, which allows you to create an binary executable. This is great for distributing a compiled binary for Windows/MacOS/Ubuntu of your selenium bot with all original source code removed, making it impossible to reverse engineer.

https://github.com/zackees/open-webdriver

**Reply by GhislainesYummyMilkies** [reply]
Wow, I wish I knew this existed before. I've been using the regular selenium webdriver this whole time.
